Components/Sensors/Inc/Data.h

```diff
@@ -36,6 +36,11 @@ typedef struct
     int32_t     pressure_1;
 } PressureTransducerData;
 
+typedef struct
+{
+    int32_t     voltage_;
```

Comment:
nit: leave a comment here noting down what the number means: `// eg. 3300 == 3.3V` (or whatever is correct)